src/main/java/Duke/Printer.java

```diff
@@ -0,0 +1,124 @@
+package Duke;
+
+public class Printer {
+    private static final String BORDER = "    ____________________________________________________________\n";
+    private static final String SPACING = "    ";
+    public static final String LOGO = " ____        _        \n"
+                                    + "|  _ \\ _   _| | _____ \n"
+                                    + "| | | | | | | |/ / _ \\\n"
+                                    + "| |_| | |_| |   <  __/\n"
+                                    + "|____/ \\__,_|_|\\_\\___|\n";
+
+    /**
+     * Prints default greeting from Duke.
+     */
+    public void printGreeting() {
+        System.out.println("Hello from\n" + LOGO);
+
+        System.out.println(BORDER + SPACING +
+                "Hello! I'm Duke\n" + SPACING +
+                "What can I do for you?\n" +
+                BORDER);
+    }
+
+    /**
+     * Prints default goodbye from Duke.
+     */
+    public void printBye() {
+        System.out.println(BORDER + SPACING + "Bye. Hope to see you again soon!\n" + BORDER);
+    }
+
+    /**
+     * Prints task added to storage.
+     *
+     * @param task Task added to storage.
+     * @param numberOfTasks Number of Tasks in storage.
+     */
+    public void  printTask(Task task, int numberOfTasks) {
+        System.out.println(BORDER
+                + SPACING
+                + "Got it. I've added this task:\n"
+                + SPACING
+                + task.toString()
+                + "\n"
+                + SPACING
+                + "Now you have " + numberOfTasks + " tasks in the list.\n"
+                + BORDER);
+    }
```

Comment:
I like how you how you formatted this portion, I can readily visualise what Duke would actually print.